### PR TITLE
Create ipywidgets.travis.yml

### PR DESCRIPTION
**Pull request by dthadi3**

### CODE REVIEW

**Review by gururajrkatti — COMMENTED** [review]
Can we also run non-python tests through yarn ? We are anyway installing firefox and starting xvfb services. Rest all LGTM

**Comment by dthadi3** [comment]
having some issues with Firefox and start the xvfb services on power, so modified the travis,yml.

File: travis-ymls/ipywidgets.travis.yml [hunk 1]
```diff
@@ -20,13 +20,6 @@ python:
     - 3.7
     - 3.8
     - 3.9
-addons:
-  apt_packages:
-    - pandoc
-  firefox: latest
-services:
-  - xvfb
```
Comment (gururajrkatti) [inline travis-ymls/ipywidgets.travis.yml]: You can manually install these using "addons/apt" and start the service using "systemctl start" are you not able to run yarn test with that ?

**Comment by seth-priya** [comment]
@dthadi3 any update on this?

**Comment by dthadi3** [comment]
@seth-priya  looking into it... will give an update or try to complete before next week.

File: travis-ymls/ipywidgets.travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #
 # Package             : ipywidgets
 # Source Repo         : https://github.com/jupyter-widgets/ipywidgets.git
-# Travis Job Link     : https://travis-ci.com/github/dthadi3/ipywidgets/builds/221085037
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/ipywidgets/builds/223543171
```
Comment (gururajrkatti) [inline travis-ymls/ipywidgets.travis.yml:5]: I don't see "tests complete" messages in ppc64le logs unlike amd64. Also I see errors like "Cannot start Firefox" in ppc64le. Pl. check

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
@dthadi3 any updates on this

File: travis-ymls/ipywidgets.travis.yml (hunk 2 above)
Comment (dthadi3) [reply travis-ymls/ipywidgets.travis.yml:5]: unable to fix the issue... can you please handover this package to your bucket or else assign to anyone.
Thanks in advance.

**Pull request closed without merging**